pkgs/development/python-modules/sapi-python-client/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, git, setuptools, setuptools_scm, fetchgit, requests, boto3, buildPythonPackage, responses }: 
+
+buildPythonPackage rec { 
+    pname = "sapi-python-client"; 
+    version = "0.1.3"; 
+
+    src = fetchgit { 
+        url = "https://github.com/keboola/sapi-python-client.git"; 
+        sha256 = "1zfxz9066ddpdmyxrwvns2v8kjn5dyyqgbiaai05k4ksrrf3zv22"; 
+        rev = "0.1.3"; 
+    }; 
```

please use github helper, pulls down just a tarball instead of git history
```suggestion
    src = fetchFromGitHub { 
        owner = "keboola";
        repo = pname;
        rev  = version;
        sha256 = "1zfxz9066ddpdmyxrwvns2v8kjn5dyyqgbiaai05k4ksrrf3zv2a"; 
    }; 
```
sha isn't right, but tha tcan be fixed